aw/ViewControllers/Reaches/Detail/RunDetailTableViewController.swift

```diff
@@ -33,8 +33,13 @@ class RunDetailTableViewController: UITableViewController {
             reach.detailUpdated == nil {
             drawView()
             print("Updating reach detail")
+            tableView.refreshControl?.beginRefreshing()
+            let refreshText = tableView.refreshControl?.attributedTitle
+            tableView.refreshControl?.attributedTitle = NSAttributedString(string: "Loading run details")
```

Comment:
I don't think we need a title on the refresh control, that's not standard in any apps I've seen.